en-US/whatsnew/whatsnew_ios_6_0.lang

```diff
@@ -0,0 +1,11 @@
+## NOTE: Those strings are displayed on the App Store in the what’s new section for Firefox for iOS 6.0 release. See https://l10n.mozilla-community.org//stores_l10n/locale/fr/apple/release/
+
+;Automatically open web links in Firefox from mail apps such as Outlook, Airmail, Mail.Ru, myMail and Spark.  Also, make one of these mail services your default mail app when sending emails from Firefox.
```

Comment:
We need to fix the double space in this string before 'Also'. Running lang_update again will fix all other locales, but you'll have to manually fix German (I see you've copied the strings 👍)